RELEASE_PROCESS.md

```diff
@@ -45,26 +45,26 @@ The following steps are what we do to release a new version of _BigchainDB Serve
    - In `bigchaindb/version.py`:
      - update `__version__` to e.g. `0.9.0` (with no `.dev` on the end)
      - update `__short_version__` to e.g. `0.9` (with no `.dev` on the end)
-   - In the docs about installing BigchainDB (and Tendermint), and in the associated scripts, recommend/install a version of Tendermint that _actually works_ with the soon-to-be-released version of BigchainDB. You can find all such references by doing a search for the previously-recommended version number, such as `0.22.8`.
+   - In the docs about installing BigchainDB (and Tendermint), and in the associated scripts, recommend/install a version of Tendermint that _actually works_ with the soon-to-be-released version of BigchainDB. You can find all such references by doing a search for the previously-recommended version number, such as 0.31.5`.
    - In `setup.py`, _maybe_ update the development status item in the `classifiers` list. For example, one allowed value is `"Development Status :: 5 - Production/Stable"`. The [allowed values are listed at pypi.python.org](https://pypi.python.org/pypi?%3Aaction=list_classifiers).
 
-1. **Wait for all the tests to pass!**
-1. Merge the pull request into the `master` branch.
-1. Go to the [bigchaindb/bigchaindb Releases page on GitHub](https://github.com/bigchaindb/bigchaindb/releases)
+2. **Wait for all the tests to pass!**
```

Comment:
Is there a reason you changed the numbering format?

Reply:
I forgot a ` in front of a number, this ended up in having a strange diff here at github. fixed it with a new commit/push

Reply:
Ahh nice, good catch! but the numbers still look the same. I mean, it really doesn't matter, but I just want to bring your attention to it, if you want to change it